packages/desktop-gui/src/app/ui-blocker.scss

```diff
@@ -0,0 +1,8 @@
+.ui-blocker {
```

Comment:
Assuming this can be used with loaders that are not fullscreen also (as mentioned in another comment), then this implementation will not work as expected. Grant it, this comment is only relevant if we do **not** have a current need for non-fullscreen loaders and therefore non-fullscreen scrims. However, I assume there is a need as `Loader` has fullscreen as a prop.